src/components/ProfileSideNav/ProfileSideNav.css

```diff
@@ -0,0 +1,18 @@
+.ProfileSideNav a {
+  display: block;
+  padding: 10px;
+  text-decoration: none;
```

Comment:
Why not tailwind on the elements itself? I assume you wanted to remove duplicate code?